docs/docs-content/clusters/edge/site-deployment/deploy-cluster.md

```diff
@@ -395,10 +425,11 @@ The next step is to use the following `docker run` command to trigger Packer bui
 - The `--env-file` option reads the **.packerenv** file.
 
 
-- The `--volume ` option mounts a local directory to our official tutorials container, `ghcr.io/spectrocloud/tutorials:1.0.8`.
+- The `--volume ` option mounts a local directory to our official tutorials container, `ghcr.io/spectrocloud/tutorials:1.0.9`.
 
+- The `sh -c "source /edge/vmware/clone_vm_template/setenv.sh && bash /edge/vmware/clone_vm_template/delete-packer-cache.sh"` shell sub-command deletes any pre-existing **packer_cache** because there is [a bug with Packer's vSphere plugin](https://github.com/hashicorp/packer-plugin-vsphere/issues/55) that makes it unable to overwrite the existing ISO file in **packer_cache**. We make sure to delete any existing cache so you don't end up accidentally using someone else's ISO file. 
```

Comment:
```suggestion
- The `sh -c "source /edge/vmware/clone_vm_template/setenv.sh && bash /edge/vmware/clone_vm_template/delete-packer-cache.sh"` shell sub-command deletes any pre-existing **packer_cache**.  A known [issue]((https://github.com/hashicorp/packer-plugin-vsphere/issues/55) with the Packer vSphere plugin causes checksum logic to ignore previous builds,  and reuse previously created ISO found in the **packer_cache** folder. The delete script removes any existing packer cache to prevent re-using a previously created ISO.
```
I cleaned up the language and the message. "We" should not be used unless we are providing a recommendation or doing something on the user's behalf. In this case, they are the ones issuing all the commands.